pkg/tls/certRenewer.go

```diff
@@ -117,8 +118,22 @@ func (c *CertRenewer) WriteCACertToSecret(caPEM *PemPair, props CertificateProps
 	logger := c.log.WithName("CAcert")
 	name := generateRootCASecretName(props)
 
-	secretUnstr, err := c.client.GetResource("", "Secret", props.Namespace, name)
+	depl, err := c.client.GetResource("", "Deployment", props.Namespace, "kyverno")
```

Comment:
The deployment name can be changed, we cannot hard code it.

Reply:
Okay, anyway to find that dynamically?

Reply:
See `KyvernoDeploymentName` in `pkg/config/config.go`.